bin/src/named.rs

```diff
@@ -635,11 +657,11 @@ fn config_https(
 }
 
 fn banner() {
-    info!("");
-    info!("    o                      o            o             ");
-    info!("    |                      |            |             ");
-    info!("  --O--  o-o  o  o  o-o  --O--  o-o   o-O  o-o   o-o  ");
-    info!("    |    |    |  |   \\     |         |  |  |  |   \\   ");
-    info!("    o    o    o--o  o-o    o          o-o  o  o  o-o  ");
-    info!("");
+    println!("");
```

Comment:
I think we want to leave this as an info line, gives folks the option of disabling it.